cmd/m3/portal.go

```diff
@@ -17,18 +17,31 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/minio/cli"
 	"github.com/minio/m3/portal"
 )
 
 // list files and folders.
 var portalCmd = cli.Command{
-	Name:   "portal",
-	Usage:  "starts portal",
-	Action: startAPIPortalCmd,
+	Name:    "service",
+	Aliases: []string{"s"},
+	Usage:   "starts m3 services, public and private APIs.",
+	Action:  startAPIServiceCmd,
```

Comment:
Can you please then update the Documentation on the README cause It still relies on portal ?

Reply:
oh lol nvm it is not there